cmd/minishift/cmd/delete.go

```diff
@@ -61,6 +60,7 @@ func runDelete(cmd *cobra.Command, args []string) {
 	}
 
 	if clearCache {
+		fmt.Printf("Deleting Minishift ISO, OpenShift container images and OpenShift origin binaries from %s/cache\n", constants.Minipath)
```

Comment:
hmm, whether you have images will depend really. I thought more about literally logging what gets deleted. The concrete file names. Not sure what @LalatenduMohanty had in mind.